packages/hardhat-core/src/internal/hardhat-network/provider/transactions/ReadOnlyValidEIP1559Transaction.ts

```diff
@@ -0,0 +1,159 @@
+import Common from "@ethereumjs/common";
+import { FeeMarketEIP1559Transaction, TxOptions } from "@ethereumjs/tx";
+import { Address, BN } from "ethereumjs-util";
+
+import {
+  FeeMarketEIP1559TxData,
+  FeeMarketEIP1559ValuesArray,
+} from "@ethereumjs/tx/src/types";
+import { InternalError } from "../../../core/providers/errors";
+
+/* eslint-disable @nomiclabs/only-hardhat-error */
+
+/**
+ * This class is like `ReadOnlyValidTransaction` but for
+ * EIP-2930 (access list) transactions.
```

Comment:
This comment seems to be wrong. There are also other comments that refer to 2930 instead of 1559.

Reply:
Done